pkg/roachprod/vm/aws/support.go

```diff
@@ -69,7 +80,8 @@ if [ "${#disks[@]}" -eq "0" ]; then
   echo "No disks mounted, creating ${mountpoint}"
   mkdir -p ${mountpoint}
   chmod 777 ${mountpoint}
-elif [ "${#disks[@]}" -eq "1" ] || [ -n "$use_multiple_disks" ]; then
+elif [ "${#disks[@]}" -eq "1" ] || [ "${#ebs_volumes[@]}" -eq "1" ] || [ -n "$use_multiple_disks" ]; then
+	# only 1 disk or 1 ebs volume - cannot select RAID'ing
```

Comment:
What about the case when the second conjunct is true, i.e., `ebs_volumes` has 1, yet there are multiple disks, i.e., `disk` has > 1?